v3/src/components/graph/components/graph.tsx

```diff
@@ -40,8 +40,8 @@ interface IProps {
 
 const marqueeState = new MarqueeState()
 
-export const Graph = observer((
-  {graphRef, enableAnimation, dotsRef, showInspector, setShowInspector}: IProps) => {
+export const Graph = observer(function Graph(
+  {graphRef, enableAnimation, dotsRef, showInspector, setShowInspector}: IProps) {
```

Comment:
Bill's current PR will conflict with this, so I think you should just back out of this change and you or Bill can come back to it later.

Reply:
I will make this change in my current PR to avoid the conflict.